db/schema_applier.hh

```diff
@@ -12,6 +12,7 @@
 #include "mutation/mutation.hh"
 #include <seastar/core/future.hh>
 #include "service/storage_proxy.hh"
+#include "replica/database.hh"
```

Comment:
Please use forward declarations instead, this one is a heavyweight.

Reply:
ok